### PR TITLE
More Info Page

### LINKED ISSUE #78: Develop More Info Page

### PR DESCRIPTION
**Pull request by malcolmrjones**
Closes #78 

Created more info page with organization info and its respective accpeted categories. Right now the address is hardcoded but the rest of the info and accepted categories comes from the backend. to see a page you navigate to `/discover/organization/:id` where :id is the ID of an organization. Once the user clicks on a search results card from the discover page they will be able to navigate to the more info page and it will open in a new tab.

**NOTE:  I have modified the prettier config a little bit to change the column width from 100 to 80**

image

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "bracketSpacing": false,
-  "printWidth": 100,
+  "printWidth": 80,
```
Comment (brucezheng) [inline .prettierrc.json:3]: I think we should do this in a separate PR.

This is difficult to review since there are so many formatting changes.

File: public/js/moreInfo.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    {{/each}}
+  </ul>
+
+  <h3>Instructions</h3>
+  <ol>
+    {{#each this.instructions}}
+      <li>{{this}}</li>
+    {{/each}}
+  </ol>
+</div>
```
Comment (brucezheng) [inline public/js/moreInfo.js:32]: Nit: Indent inside of a control statement:

```
{{#each foo}}
  <div></div>
{{/each}}
```

File: .prettierrc.json (hunk 1 above)
Comment (malcolmrjones) [reply .prettierrc.json]: Sounds good!

Comment (malcolmrjones) [reply .prettierrc.json]: Sounds good!
